src/commands/debugger/logs/list.js

```diff
@@ -92,7 +92,7 @@ class DebuggerLogsList extends TwilioClientCommand {
 
 DebuggerLogsList.description = `Show a list of log events generated for the account
 
-Argg, this is only a subset of the log events and live tailing isn't quite ready! Think this is a killer feature? Email aroach@twilio.com and tell him.`;
+Argg, this is only a subset of the log events and live tailing isn't quite ready! Think this is a killer feature? Let us know https://bit.ly/twilio-cli-feedback`;
```

Comment:
_Let us know here: https://bit.ly/twilio-cli-feedback_  ?

What does @aroach think?

Reply:
yeah, @childish-sambino, that tweak looks OK to me.